Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -81,6 +87,7 @@ public extension XCScheme {
 
         // MARK: - Init
 
+        @available(*, deprecated, message: "Use the init() that consolidates pathRunnable and runnable into a single parameter.")
```

Comment:
@pepicrft I think this deprecation warning was attached to the wrong init since the other convenience init now only accepts a PathRunnable